source/gui/settingsDialogs.py

```diff
@@ -4428,6 +4428,15 @@ def makeSettings(self, settingsSizer):
 		)
 		self.bindHelpEvent("BrailleSettingsInterruptSpeech", self.brailleInterruptSpeechCombo)
 
+		self.formattingDisplayCombo = sHelper.addLabeledControl(
+			# Translators: This is a label for a combo-box in the Braille settings panel.
+			labelText=_("Formatting display"),
+			wxCtrlClass=nvdaControls.FeatureFlagCombo,
+			keyPath=("braille", "fontAttributeDisplay"),
+			conf=config.conf,
+		)
+		self.bindHelpEvent("BrailleSettingsFormattingDisplay", self.formattingDisplayCombo)
```

Comment:
Anchor name without category name (see comments in the User Guide):

```suggestion
		self.bindHelpEvent("BrailleFormattingDisplay", self.formattingDisplayCombo)
```